### PR TITLE
Corrected Grammer Mistakes

### PR DESCRIPTION
**Pull request by gargarchit**
Corrected Some Grammer Mistake now docs look much better to read

### Full changelog

* [Edited docs too...]
* [Added cool new feature ...]
* ...

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] [Adheres to Kong style guide](https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md)
- [x] Spellchecked my updates
- [x] Tagged "Team Docs" as reviewers
- [x] Ready to be merged

### CODE REVIEW

**Comment by gargarchit** [comment]
Please Merge this

File: templates/how-to.md [hunk 1]
```diff
@@ -14,15 +14,14 @@
...
-accomplish by following the steps. 
+This is an introduction to this how-to guide. It should be limited to a few
+succinct sentences that introduce what the user will be learning to do. A
+The how-to guide should provide an end-goal or something that the user will accomplish by following the steps. 
 
 ### Prerequisites
 
 - A bulleted list of everything a user needs to complete the How-to guide
-- May be broken into multiple sections for additional clarity
+- Maybe broken into multiple sections for additional clarity
```
Comment (levfishbluefish) [inline templates/how-to.md]: The original usage is correct, https://dictionary.cambridge.org/us/grammar/british-grammar/maybe-or-may-be

File: templates/how-to.md [hunk 2]
```diff
@@ -14,15 +14,14 @@
 
 ### Introduction
 
-This is an introduction to this how to guide. It should be limited to a few
-succinct sentences that introduces what the user will be learning to do. A
-How-to guide should provide an end-goal, or something that the user will 
-accomplish by following the steps. 
+This is an introduction to this how-to guide. It should be limited to a few
```
Comment (levfishbluefish) [inline templates/how-to.md]:
```suggestion
Use this section to provide an introduction to the how-to guide and provide the user with context for the guide. It should be limited to a few succinct sentences and provide a clear end-goal or outline of what the user will accomplish by following the guide
```

File: README.md [hunk 3]
```diff
@@ -74,11 +74,11 @@ scraper to test out config changes.
...
 
 - Make sure that the `resty` and `luajit` executables are in your `$PATH` (installing kong should install them)
-- Several Lua rocks are needed. Easiest way to get all of them is to execute `make dev` in the Kong folder
+- Several Lua rocks are needed. The easiest way to get all of them is to execute `make dev` in the Kong folder
 - Have a local clone of Kong
-- In the Kong repository, checkout the desired branch/tag/release
+- In the Kong repository, check out the desired branch/tag/release
 - To generate the Admin API docs:
-  - Run: `KONG_PATH=path/to/your/kong/folder KONG_VERSION=0.14.x gulp admin-api-docs`
+  - Run: `KONG_PATH=path/to/your/kong/folder KONG_VERSION=0.14.x gulp admin-API-docs`
```
Comment (levfishbluefish) [inline README.md]: This is an unnecessary capitalization

**Comment by gargarchit** [comment]
Done

**Comment by levfishbluefish** [comment]
Merge the two suggestions above and I will approve and merge this PR

Thanks!

**Comment by gargarchit** [comment]
I merged the two above Suggestions
Thanks for reviewing this!

**Pull request merged**